pkgs/applications/office/wordgrinder/default.nix

```diff
@@ -18,6 +18,10 @@ stdenv.mkDerivation rec {
     "LUA_LIB=${lua52Packages.lua}/lib/liblua.so"
   ] ++ stdenv.lib.optional stdenv.isLinux "XFT_PACKAGE=--libs=\{-lX11 -lXft\}";
```

Comment:
```suggestion
   ];
```

Reply:
We already have this in `preRebuild`, so no need to duplicate it.

Reply:
hang on...